src/Microsoft.DotNet.Build.Tasks/Targets/resources.targets

```diff
@@ -27,12 +27,21 @@
         ResxFilePath="$(StringResourcesPath)" 
         IntermediateFilePath="$(IntermediateResOutputFileFullPath)" 
         OutputSourceFilePath="$(ResourceSRPath)" 
-        AssemblyName="$(AssemblyName)" >
+        AssemblyName="$(AssemblyName)" 
+        ContinueOnError="ErrorAndStop" >
+        
+        <Output PropertyName="GenerateResourcesCodeOutputMessage" TaskParameter="Message" />
```

Comment:
@weshaggard  Log.LogError didn't work at least when compiled on TFS side. I am not sure if we have Log.Error. when used Log.LogError I am still getting the behavior we have seen before that the build break with some other errors.